SceneComponent.js

```diff
@@ -0,0 +1,21 @@
+const React = require('react');
+const ReactNative = require('react-native');
+const {Component, } = React;
+const {View, StyleSheet, } = ReactNative;
+
+const StaticContainer = require('react-static-container');
+
+class SceneComponent extends Component {
```

Comment:
stateless component would be awesome here like this one https://github.com/skv-headless/react-native-scrollable-tab-view/blob/master/Button.ios.js